examples/image_segmentation_DIS/android/app/src/main/java/com/example/segmentationdis/ImageSegmentationHelper.kt

```diff
@@ -0,0 +1,185 @@
+                        .add(NormalizeOp(0f, 1f))
+                        .build()
+
+                val tensorImageFromBitmap = TensorImage.fromBitmap(bitmap)
+                // Preprocess the image and convert it into a TensorImage for segmentation.
+                val tensorImageWithImageProcessor = imageProcessor.process(tensorImageFromBitmap)
+                val bitmapOutput = getBitmap(tensorImageWithImageProcessor)
+                val bitmapResult = cropBitmap(bitmapOutput, percent)
+                val inferenceTime = SystemClock.uptimeMillis() - startTime
+                if (isActive) {
+                    _segmentation.emit(SegmentationResult(bitmapResult, inferenceTime))
+                }
+            }
+        } catch (e: Exception) {
+            Log.i(TAG, "Image segment error occurred: ${e.message}")
+            _error.emit(e)
+        }
+    }
+
+    /**
+     * Create output and run model
+     */
+    private fun getBitmap(tensorImage: TensorImage): Bitmap {
+        val (_, h, w, c) = interpreter!!.getOutputTensor(0).shape()
+        val outputBuffer = FloatBuffer.allocate(h * w * c)
+        val inputBuffer = TensorImage.createFrom(tensorImage, DataType.FLOAT32).buffer
+
+        outputBuffer.rewind()
+        interpreter?.run(inputBuffer, outputBuffer)
+        outputBuffer.rewind()
+
+        val array32 = FloatArray(h * w * c)
+        outputBuffer.get(array32)
+
+        return floatArrayToBitmap(array32, w, h)
+    }
+
+    /**
+     * Convert float array to bitmap
+     */
+    private fun floatArrayToBitmap(floatArray: FloatArray, width: Int, height: Int): Bitmap {
+        val bitmap = Bitmap.createBitmap(width, height, Bitmap.Config.ARGB_8888)
+
+        for (h in 0 until height) {
+            for (w in 0 until width) {
+                // Get the index in the float array
+                val index = h * width + w
+
+                // Get the float value, ensuring it's between 0.0 and 1.0
+                val value = floatArray[index].coerceIn(0.0f, 1.0f)
+
+                // Set gray value
+                val grayValue = (value * 255).toInt()
+                val alpha = if (value > 0.95f) 0 else 255
+
+                val color =
+                    Color.argb(alpha, grayValue, grayValue, grayValue)
+                bitmap.setPixel(w, h, color)
+            }
+        }
+
+        return bitmap
+    }
+
+    /**
+     * Crop bitmap before emit result
+     */
+    private fun cropBitmap(bitmap: Bitmap, percent: Float): Bitmap {
+        return Bitmap.createBitmap(
+            bitmap,
+            ((1 - percent) / 2 * bitmap.width).toInt(),
+            0,
+            (bitmap.width * percent).toInt(),
+            bitmap.height
+        )
+    }
+
+    enum class Delegate {
+        CPU, NNAPI
+    }
+
+    enum class Model(val title: String, val fileName: String) {
+        ISNET_TFL_DRQ("isnet_1", "isnet_tfl_drq.tflite"),
```

Comment:
This only needs isnet_tfl_drq.tflite, you can remove the selector for the other two models.